### DISCUSSION TITLE
How do I remove the pathway row from my oncoplot?

### DISCUSSION THREAD

**Discussion by Sjs3942** [PoisonAlien/maftools#984] [Q&A]
Hi, 

I am trying to create an oncoplot of genes that are mutated within a pathway I defined. The code is working great but does anyone know how to remove the bottom grey row of the oncoplot that shows the distribution of the pathway? 

`#Define the pathway
GI_R_pathways = data.frame(
  Genes = c(
    "ATM",
    "BRCA2", 
    "ATR",
    "BRCA1", 
    "CHEK2",
    "ERCC2",
    "MSH6",
    "POLQ", 
    "STAG2", 
    "PPM1D", 
    "SMC1A"
  ),
  Pathway = rep(c(
    "Genome_integrity"
  ), c(11)),
  stringsAsFactors = FALSE
)

oncoplot(maf = maf.filef, pathways = GI_R_pathways)`

<img width="351" alt="Screenshot 2023-11-02 130957" src="https://github.com/PoisonAlien/maftools/assets/108011517/6fdd085f-1b1f-4e81-a046-1b7acd5d48b8">

**Comment by PoisonAlien** [comment]
Hi,
You could just draw an `oncoplot` with `genes` argument

```r
maftools::oncoplot(maf = maf.filef, genes = c("POLQ", "ATR", "BRCA1", "ERCC2"))
```

**Reply by Sjs3942** [reply]
Yes, and I have done that before, but I was hoping to find a method that was more automated than manually inputting the genes since I'm making a lot of oncoplots for different combinations of samples and pathways. @PoisonAlien
